src/components/TodoItem/TodoItem.jsx

```diff
@@ -73,8 +35,10 @@ const TodoItem = ({ text, createdAt, taskId, getTasks, isDone }) => {
         <input
           className="isDone-checkbox"
           type="checkbox"
-          onChange={updateStatus}
-          checked={isDone}
+          onChange={() => {
+            mutationUpdate.mutate({ taskId, isDone: !isDone });
+          }}
+          defaultChecked={isDone}
```

Comment:
`useUpdateStatus` returns an object that has `isLoading` property, and it's better to pass this value to the disabled prop of this checkbox input, because when the user click on the checkbox, the api has been called, but if user click on the checkbox immediately, previous api hasn't success yet, so pass this property to disabled prop